### DISCUSSION TITLE
Island Schematic Paste Offest

### DISCUSSION THREAD

**Discussion by Tyguh** [BG-Software-LLC/SuperiorSkyblock2#1461] [Pending Feature Requests] [answered]
Hey, would it be possible for you to implement a way to set the offset of the island border from a schematic's spawn point? This would be useful for schematics that don't have a centered spawn point as they end up being pasted outside the border.

**Comment by Lemvs** [comment, marked as answer]
When saving a schematic, you should stand in its center so that it fits within the border, however, you can move the player's spawn location using the spawn-offset option in menus/island-creation.yml.

@OmerBenGera you can mark this answered and close it.
